wiki/Skin/en.md

```diff
@@ -4,7 +4,29 @@ stub: true
 
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned." 
 
-A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned."
+## Reasons to Use Skins
+
+- Skins can be used to further optimize a player's gameplay, since most skins do away with showing that a player has hit a note perfectly (to only show the minimum amount of information to a player in-game). 
+- Another gameplay-improving feautre that skins provide (for osu! standrad) is chagning the cursor style and trail. This can be especially helpful if a player finds the default skin cursor intruding into the action.
+
+## How to Use Custom Skins
+
+Skins can either be created from scratch (see [Skinning](/wiki/Skinning) for creating your own skin), or downloaded as a ".osk" file and imported into the game. To download and use an osu! skin you have to:
+
+1. Pick one of the [Popular Skin Websites](#popular-skin-websites) and open the link.
+2. Choose a skin that suits your tastes by clicking on it's name or picture.
+3. Download the skin's ".osk" from the opened skin's page.
+4. Either double click the ".osk" file to automatically import and select it as default skin in osu! (the game may take a bit to re-open so just wait for a little for it to load the new skin).
+
+## Popular Skin Websites
+
+- [Circle People Skins](https://circle-people.com/skins/)
```

Comment:
not sure about naming it "Circle People Skins", doesn't convey really what it is. This is a collection of top-player skins from the times that circle people was active, aka its heavily outdated by now